docs/site/Loopback-component-authentication.md

```diff
@@ -0,0 +1,611 @@
+---
+lang: en
+title: 'Authentication'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Loopback-component-authentication.html
+---
+
+## Overview
+
+Security is of paramount importance when developing a web or mobile application
+and usually consists of two distinct pieces:
+
+- Authentication
+- Authorization
+
+`Authentication` is a process of verifying someone's identity before a protected
+endpoint is accessed.
+
+`Authorization` is a process of verifying the actions that an authenticated user
+is allowed to perform on a protected endpoint.
+
+{% include note.html content="
+For a description of an `Authorization` process, please see [Authorization](Loopback-component-authorization.md).
+" %}
+
+This document describes the details of the LoopBack 4 `Authentication`
+component.
+
+Here is a **high level** overview of the authentication component.
+
+![authentication_overview_highlevel](./imgs/authentication_overview_highlevel.png)
```

Comment:
Let's crop the diagram so that there is no blank area to the left:

![image](https://user-images.githubusercontent.com/540892/59291243-432ec180-8c2f-11e9-8705-e783e76739aa.png)